### DISCUSSION TITLE
Escaping special characters?

### DISCUSSION THREAD

**Discussion by ztronzo** [Palakis/obs-websocket#900] [Q&A] [answered]
I am trying to update a browser-source url with `https://mywebsite/?parameter=something` using
/command="SetSourceSettings",sourceName="BrowserTest2",sourceSettings=url="https://mywebsite/?parameter=something"

as a result I get everything truncated after and including the "=" character 
    "url": "https://mywebspite/?parameter",

Are there any escape characters I can use or a method for inputting the equal sign "=" character (DEC: 61, HEX :3d) ?

**Comment by tt2468** [comment, marked as answer]
Please use obscommand's support system. This is not the right place.

**Reply by ztronzo** [reply]
thanks for your prompt response, you can delete/remove this discussion
